webview/src/experiments/components/App.test.tsx

```diff
@@ -183,46 +183,69 @@ describe('App', () => {
     await expectHeaders(['A', 'C', 'D', 'B'])
   })
 
-  it('should add a "branch/tags" column if the table is sorted', () => {
-    renderTable(sortedTableStateFixture)
+  describe('Sorted (Flattened) Table', () => {
+    it('should add a "branch/tags" column', () => {
+      renderTable(sortedTableStateFixture)
 
-    const branchHeader = screen.getByTestId('header-branch')
-    expect(branchHeader).toBeInTheDocument()
+      const branchHeader = screen.getByTestId('header-branch')
+      expect(branchHeader).toBeInTheDocument()
 
-    const branchHeaderTextContent =
-      within(branchHeader).getByText('Branch/Tags')
-    expect(branchHeader).toBeInTheDocument()
+      const branchHeaderTextContent =
+        within(branchHeader).getByText('Branch/Tags')
+      expect(branchHeader).toBeInTheDocument()
 
-    fireEvent.mouseEnter(branchHeaderTextContent, { bubbles: true })
-    expect(screen.getByRole('tooltip')).toBeInTheDocument()
-    expect(screen.getByRole('tooltip')).toHaveTextContent(
-      'The table has limited functionality while sorted. Clear all sorts to have nested rows and increase/decrease commits.'
-    )
+      fireEvent.mouseEnter(branchHeaderTextContent, { bubbles: true })
+      expect(screen.getByRole('tooltip')).toBeInTheDocument()
+      expect(screen.getByRole('tooltip')).toHaveTextContent(
+        'The table has limited functionality while sorted. Clear all sorts to have nested rows and increase/decrease commits.'
+      )
 
-    expect(screen.getByTestId('branch___main').textContent).toStrictEqual(
-      'main'
-    )
-  })
+      const branchCell = screen.getByTestId('branch___main')
+      expect(branchCell).toHaveTextContent('main')
+    })
 
-  it('should add a "parent" column if the table is sorted', () => {
-    renderTable(sortedTableStateFixture)
+    it('should show two branches in the "branch/tags" column cell if the row belongs to two branches', () => {
+      renderTable(sortedTableStateFixture)
 
-    const commitHeader = screen.getByTestId('header-commit')
-    expect(commitHeader).toBeInTheDocument()
+      const cellBranches = within(
+        screen.getByTestId('branch___other-branch')
+      ).getAllByRole('listitem')
 
-    const commitHeaderTextContent = within(commitHeader).getByText('Parent')
-    expect(commitHeader).toBeInTheDocument()
+      expect(cellBranches[0]).toHaveTextContent('main')
+      expect(cellBranches[1]).toHaveTextContent('other-branch')
```

Comment:
Despite tooltips working locally and in storybook, I can't get a tooltip test to pass:

```js
      fireEvent.mouseEnter(cellBranches[0], { bubbles: true })

      advanceTimersByTime(NORMAL_TOOLTIP_DELAY[0])

      expect(screen.getByRole('tooltip')).toBeInTheDocument() // fails with cant find role error
```

Tried targeting a different element or adjusting the timer amount, but same error. Am I missing something?

Reply:
Are you using fake timers? 

```
beforeAll(() => {
      jest.useFakeTimers()
    })
    afterAll(() => {
      jest.useRealTimers()
    })
    ```
   I know there are some in the "Tooltips" describe block, but not sure about this one.

Reply:
Whoops, forgot to add that 😄. Thanks for catching that! I'll add tooltip tests.